src/app/components/search/SearchResultsTable/ItemThumbnail.js

```diff
@@ -32,13 +33,22 @@ const ItemThumbnail = ({
       mediaType = mediaTypeFromUrl(url);
     }
     return (
-      <div className={`${styles.thumbnail} ${styles.container}`}>
-        <div className={styles.iconContainer}>
-          <Tooltip title={mediaType} arrow>
+      <Tooltip
+        title={
+          <FormattedMessage
+            id="ItemThumbnail.mediaType"
+            defaultMessage="{mediaType}"
+            description="type of the media item"
+            values={{ mediaType }}
+          />
+        }
+      >
```

Comment:
@danielevalverde , this still doesn't fix the problem - we actually need to map each value of `mediaType` to a different localizable string, like we do here: https://github.com/meedan/check-web/blob/develop/src/app/components/media/MediaTypeDisplayName.js. Actually, I think we can just use that component here :)